packages/upload-media/README.md

```diff
@@ -0,0 +1,108 @@
+# `@wordpress/upload-media`
```

Comment:
I have been thinking about whether we need a new package or whether we can reuse the existing `media-utils`.

The problem right now is that `media-utils` is tied to WordPress (apiFetch call) and `upload-media` is basically a media upload handler, queue system that is independent from WordPress. Conceptually it can be seen as a "sub package" of block-editor. The reason it's separate is that it might be useful to handle uploads in a similar way outside the block editor. So I think it's probably the right way to go.

Can we document this? I mean expand the description of the package a little bit.

Reply:
Sure, I can expand the documentation, both now and in in follow-up PRs.